### PR TITLE
Release 3.8.1

### PR DESCRIPTION
**Pull request by sloloris**

### CODE REVIEW

File: spec/info.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@ contact:
 license:
   name: Apache 2.0
   url: http://www.apache.org/licenses/LICENSE-2.0.html
-version: 3.7.1
+version: 3.8.1
```
Comment (ldhenry) [inline spec/info.yaml]: I think this should be version 3.8.0
```suggestion
version: 3.8.0
```

**Review by ldhenry — CHANGES_REQUESTED** [review]
Looks good aside from the version number.

File: spec/info.yaml (hunk 1 above)
Comment (sloloris) [reply spec/info.yaml]: crap

**Review by ldhenry — CHANGES_REQUESTED** [review]
Unfortunately I believe you'll have to change the version in the description and change the branch name as well. It might be easier to open a new PR

**Comment by sloloris** [comment]
wrong version oops

**Pull request closed without merging**